server/routes/api/moderator/moderatorHandler.ts

```diff
@@ -8,11 +15,32 @@ export default class Handler {
     this.dbHandler = new DBHandler();
   }
 
+  requestValidation = async (requestId: number, newStatus: string): Promise<Validation> => {
+    const contactRequest = await this.dbHandler.getContactRequest(requestId);
+    if (!contactRequest.length) {
+      return { isValid: false, message: "Incorrect id" };
+    }
+
+    if (!newStatus) {
```

Comment:
First check if newStatus is undefined and then check the request id cause there you query the db